BlueDBOnDisk/src/test/java/org/bluedb/disk/collection/CollectionMetaDataTest.java

```diff
@@ -74,13 +74,13 @@ public void test_getAndAddToSerializedClassList() throws Exception {
 		Class<? extends Serializable>[] testValue1PlusDefaults = concatenate(defaultClasses, testValue1);
 		Class<? extends Serializable>[] testValueBothPlusDefaults =  concatenate(defaultClasses, testValueBoth);
 
-		Class<? extends Serializable>[] afterAdding1 = metaData.getAndAddToSerializedClassList(TestValue.class);
+		Class<? extends Serializable>[] afterAdding1 = metaData.getAndAddToSerializedClassList(TestValue.class, Arrays.asList());
 		assertArrayEquals(testValue1PlusDefaults, afterAdding1);
 
-		Class<? extends Serializable>[] afterAdding1Again = metaData.getAndAddToSerializedClassList(TestValue.class);
+		Class<? extends Serializable>[] afterAdding1Again = metaData.getAndAddToSerializedClassList(TestValue.class, Arrays.asList());
 		assertArrayEquals(testValue1PlusDefaults, afterAdding1Again);
 
-		Class<? extends Serializable>[] afterAddingBoth = metaData.getAndAddToSerializedClassList(TestValue.class, TestValue2.class);
+		Class<? extends Serializable>[] afterAddingBoth = metaData.getAndAddToSerializedClassList(TestValue.class, Arrays.asList(TestValue2.class));

```

Comment:
Do we have tests using the builder instead of the deprecated method? I'd rather update most stuff to use the builder at some point. Doesn't have to be now.